pkg/engine2/testdata/vpc_import_to_lambda.dataflow-viz.yaml

```diff
@@ -4,6 +4,7 @@ resources:
     children:
         - aws:security_group:vpc:lambda_function-security_group
         - aws:subnet:vpc:subnet1
+        - aws:subnet:vpc:subnet2
```

Comment:
I assume this changed due to the addition of the namespace (shown in the `.expect.yaml`)